.travis.yml

```diff
@@ -13,4 +14,4 @@ rvm:
   - truffleruby
 
 install: ( cd api && bundle install --jobs=3 --retry=3 )
-script: ( cd $TEST_DIR && rake )
+script: ( if [ $TEST_DIR=='opentracingshim' ]; then gem install opentracing -v 0.5.0; fi && cd $TEST_DIR && rake )
```

Comment:
This is not good, but gets around the we aren't really installing `sdk` or `opentracingshim` issue (since opentracingshim depends on opentracing)